Homework 33/task_33.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+
+const subjects = {
+    mathematics: {
+        students: 200,
+        teachers: 6
+    },
+    biology: {
+        students: 120,
+        teachers: 6
+    },
+    geography: {
+        students: 60,
+        teachers: 2
+    },
+    chemistry: {
+        students: 100,
+        teachers: 3
+    }
+}
+
+// Task 1
+const lessonNames = Object.keys(subjects).join();
+console.log('Task 1: ', lessonNames);
+
+// Task 2
+let total = 0;
+for (const key in subjects) {
+    for (const key2 in subjects[key]) {
+        total += subjects[key][key2];
```

Comment:
(for all tasks) better not to use nested loops; instead concrete field should be processed:

```
let total = 0;
for (const { students, teachers } of Object.values(subjects)) {
    total += (students + teachers);
}
```

also, `.reduce` as an option:

```
const total = Object.values(subjects).reduce((acc, { students, teachers }) => acc += (students + teachers), 0)
```